src/test/kotlin/lotto/model/LottoCountTest.kt

```diff
@@ -0,0 +1,47 @@
+package lotto.model
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import java.lang.IllegalArgumentException
+
+class LottoCountTest {
+
+    @Test
+    @DisplayName("수동과 자동 로또 개수의 범위 확인")
+    fun `check lotto count validation`() {
+        val price = Price(5000)
```

Comment:
```suggestion
        val price = Price(5_000)
```
천 단위에 언더바를 표시하면 가독성이 좋아져요